tests/phpunit/Utilities/Plugin_Request_Utility_Tests.php

```diff
@@ -75,19 +78,21 @@ public function test_initialize_runner_with_ajax() {
 	public function test_destroy_runner_with_cli() {
 		global $wpdb, $table_prefix, $wp_actions;
 
+		$this->set_up_mock_filesystem();
```

Comment:
Similar to above, why does this need to be called here?

Reply:
@felixarntz we prepare the runner in these tests, so we use the mock filesystem to prevent errors in the GitHub action (https://github.com/10up/plugin-check/pull/173#discussion_r1206553708)